wordpress-mobile/WordPress-iOS
# iOS 14 All Time Widget

Fixes #NA

Adds the "All Time" widget to the iOS 14 Stats widgets (see example below)

<p align=center>
<img height=450 src=https://user-images.githubusercontent.com/34376330/106681046-540b1400-6585-11eb-80a3-2fba2eed757a.PNG>
</p>

To test:

- build/run and make sure that you can install both "Today" and "All Time" widgets
- Trigger some stats updates, open the app, check those stats and make sure they are the same in the widgets
- Trigger other stats updates, and without opening the app, make sure they are reflected in the widgets when the timeline updates

tip: the normal timeline update is 1 hour; so either start the review, then have lunch, a workout or a nap and come back, or, if you don't want to wait that long, you can tweak the properties `refreshInterval = 5` and `minElapsedTimeToRefresh = 1`in `SiteListProvider.swift` to speed up things (lower values would not work).

PR submission checklist:
